package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -154,48 +187,54 @@ class WatsonCrickDist(NucPairDist):
     their index in the lists given as arguments.
 
     Parameters
-    __________
+    ----------
     strand1: List[Residue]
         First list of bases
     strand2: List[Residue]
         Second list of bases
     n1_name: str (optional)
-        Name of Nitrogen 1 of nucleic acids
-        by default assigned to N1
+        Name of Nitrogen 1 of nucleic acids, by default assigned to N1
     n3_name: str (optional)
-        Name of Nitrogen 3 of nucleic acids
-        by default assigned to N3
+        Name of Nitrogen 3 of nucleic acids, by default assigned to N3
     g_name: str (optional)
-        Name of Guanine in topology
-        by default assigned to G
+        Name of Guanine in topology, by default assigned to G
     a_name: str (optional)
-        Name of Adenine in topology
-        by default assigned to G
+        Name of Adenine in topology, by default assigned to G
```

Comment:
As a minor typo comment, I would assume this is meant to read "...assigned to A" (but otherwise lgtm :) )

Reply:
Should be addressed now thanks!